doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -144,61 +144,129 @@ presentation of Cic is given in Chapter :ref:`calculusofinductiveconstructions`.
+                          : `one_decl_notation`
+   one_decl_notation : `string` := `constr` `ident_opt3`
+   constr : `term1`
+          : @ `qualid` `instance`
+   ident_opt3 : : `ident`
+              : `empty`
+   record_binder : `name`
+                 : `name` `record_binder_body`
+   record_binder_body : `binder_list_opt` `of_type_with_opt_coercion` `term`
+                      : `binder_list_opt` `of_type_with_opt_coercion` `term` := `term`
+                      : `binder_list_opt` := `term`
+   of_type_with_opt_coercion : :>>
+                             : :> >
+                             : :>
+                             : : > >
+                             : : >
+                             : :
+   num_opt2 : | `num`
+            : `empty`
+   quoted_attributes_list_opt : `quoted_attributes_list_opt` #[ `attribute_list_comma_opt` ]
+                              : `empty`
+   attribute_list_comma_opt : `attribute_list_comma`
+                            : `empty`
+   attribute_list_comma : `attribute_list_comma` , `attribute`
+                        : `attribute`
+   attribute : `ident` `attribute_value`
+   attribute_value : = `string`
+                   : ( `attribute_list_comma_opt` )
+                   : `empty`
+   typeclass_constraint_list_comma : `typeclass_constraint_list_comma` , `typeclass_constraint`
+                                   : `typeclass_constraint`
+   typeclass_constraint : ! `term`
+                        : { `name` } : `exclam_opt` `term`
+                        : `name_colon` `exclam_opt` `term`
+                        : `term`
+   exclam_opt : !
+              : `empty`
+   of_alt : of
+          : &
+   atomic_constr : `qualid` `instance`
+                 : `sort`
+                 : `numeral`
+                 : `string`
+                 : _
+                 : ? [ `ident` ]
+                 : ? [ ? `ident` ]
+                 : ? `ident` `evar_instance`
+   evar_instance : @{ `inst_list_semi` }
+                 : `empty`
+   inst_list_semi : `inst_list_semi` ; `inst`
+                  : `inst`
+   inst : `ident` := `term`
+   instance : @{ `universe_level_list_opt` }
+            : `empty`
+   universe_level_list_opt : `universe_level_list_opt` `universe_level`
+                           : `empty`
+   universe_level : Set
+                  : Prop
+                  : Type
+                  : _
+                  : `qualid`
+   binders_fixannot : { `ident` `impl_ident_tail` `binders_fixannot`
+                    : `fixannot`
+                    : `binder` `binders_fixannot`
+                    : `empty`
+   impl_ident_tail : }
+                   : `name_list` : `term` }
+                   : `name_list` }
+                   : : `term` }
+   fixannot : { struct `ident` }
+            : { wf `constr` `ident` }
+            : { measure `constr` `ident_opt` `constr_opt` }
+            : { struct `name` }
+            : `empty`
```

Comment:
Not in the right place.

Reply:
You want to put that right after `fix_body                        ::=  ident binders_fixannot type_cstr := term`?

Reply:
Something like this.

Reply:
done